installCentOS.sh

```diff
@@ -2,6 +2,7 @@ sudo yum group install -y  "Development Tools"
 sudo yum install -y wget
 sudo yum install -y boost-devel
 sudo yum install -y python3
+sudo apt-get install -y mafft
```

Comment:
I think CentOS uses yum commands. I can check for you if `sudo yum install -y mafft` would work

Reply:
Also, do you know how to get this installed on MacOS (for the installMacOS.sh script?